Extension/package.json

```diff
@@ -282,2546 +282,2593 @@
+                "readability-*",
+                "readability-avoid-const-params-in-decls",
+                "readability-braces-around-statements",
+                "readability-const-return-type",
+                "readability-container-*",
+                "readability-container-contains",
+                "readability-container-data-pointer",
+                "readability-container-size-empty",
+                "readability-convert-member-functions-to-static",
+                "readability-delete-null-pointer",
+                "readability-duplicate-include",
+                "readability-else-after-return",
+                "readability-function-*",
+                "readability-function-cognitive-complexity",
+                "readability-function-size",
+                "readability-identifier-*",
+                "readability-identifier-length",
+                "readability-identifier-naming",
+                "readability-implicit-bool-conversion",
+                "readability-inconsistent-declaration-parameter-name",
+                "readability-isolate-declaration",
+                "readability-magic-numbers",
+                "readability-make-member-function-const",
+                "readability-misleading-indentation",
+                "readability-misplaced-array-index",
+                "readability-named-parameter",
+                "readability-non-const-parameter",
+                "readability-qualified-auto",
+                "readability-redundant-*",
+                "readability-redundant-access-specifiers",
+                "readability-redundant-control-flow",
+                "readability-redundant-declaration",
+                "readability-redundant-function-ptr-dereference",
+                "readability-redundant-member-init",
+                "readability-redundant-preprocessor",
+                "readability-redundant-smartptr-get",
+                "readability-redundant-string-*",
+                "readability-redundant-string-cstr",
+                "readability-redundant-string-init",
+                "readability-simplify-*",
+                "readability-simplify-boolean-expr",
+                "readability-simplify-subscript-expr",
+                "readability-static-*",
+                "readability-static-accessed-through-instance",
+                "readability-static-definition-in-anonymous-namespace",
+                "readability-string-compare",
+                "readability-suspicious-call-argument",
+                "readability-uniqueptr-delete-release",
+                "readability-uppercase-literal-suffix",
+                "readability-use-anyofallof",
+                "zircon-temporary-objects"
+              ]
+            },
+            "uniqueItems": true,
+            "markdownDescription": "%c_cpp.configuration.codeAnalysis.clangTidy.checks.disabled.markdownDescription%",
+            "scope": "resource"
+          }
+        }
+      },
+      {
+        "title": "VcFormat",
```

Comment:
I would prefer to have this header be "Formatting" and include the settings for clang_format in it.